Tests/Asm/Ansible/AnsibleTest.php

```diff
@@ -68,10 +68,11 @@ public function testAnsibleCommandNotFoundException()
      */
     public function testAnsibleNoCommandGivenException()
     {
-        $this->markTestIncomplete();
-        new Ansible(
-            $this->getProjectUri()
-        );
+        // TODO: Not sure why the following command should give an error.
```

Comment:
Until there is a solution for the erroring, the test should not be successful; I'd expect an incomplete or no test - otherwise "oh, cool, all tests green" even if there is a possible regression here.

Reply:
Hi @maschmann,

basically I was trying to figure out why you are expecting for an exception from that code because, by my understanding, the just instances Ansible with a working path. In other words, what is the unexpected/problematic condition?

Thanks,
Adriano